dts/msm8916/msm8916-mtp.dts

```diff
@@ -39,4 +39,13 @@
 		compatible = "asus,z010d", "qcom,msm8916", "lk2nd,device";
 		lk2nd,match-cmdline = "* mdss_mdp.panel=1:dsi:0:qcom,mdss_dsi_r69339_720p_video:1:none";
 	};
+
+	asus-p024 {
+		model = "ASUS ZenPad 8.0";
+		compatible = "asus,p024", "qcom,msm8916", "lk2nd,device";
+		lk2nd,match-cmdline = "* mdss_mdp.panel=1:dsi:0:qcom,mdss_dsi_rm27013_800x1280_video:1:none*";
```

Comment:
Your device seems to have multiple panels. Please add all of them with a `panel` node similar to the alto5-vdf above. Then use the generic `asus,p024-panel` compatible in the Linux device tree, it will automatically pick the correct panel driver.